docs/rackhd/configuration.rst

```diff
@@ -69,6 +71,9 @@ httpsBindPort          Local port to use for HTTPS. Typically, port 443.
 httpsCert              Filename of the X.509 certificate to use for TLS. Expected format is PEM.
 httpFileServiceRoot    Directory path for for storing uploaded files on disk.
 httpFileServiceType    Backend storage mechanism for file service. Currently only FileSystem is supported.
+httpProxies            Optional http proxies list. "localPath"/"remotePath" are optional ( defaults to "/").
+                       "server" is a must for proxy, both http and https servers are supported.
+                       http://<RackHD_Server_IP>:8080/localPath folder will be mapped to <server>/remotePath with httpProxies.
```

Comment:
It might be better to phrase this as:

    Proxied servers will be available at the local directory you specify to the servers running workflows. For example, { "server": "http://centos.eecs.wsu.edu", "localPath": "centos" } would map requests to /centos/ to http://centos.eecs.wsu.edu/